bootstrap/templates/new/mix.exs

```diff
@@ -20,7 +20,8 @@ defmodule <%= app_module %>.Mixfile do
      config_path: "../../config/config.exs",
      lockfile: "../../mix.lock",<% else %>
```

Comment:
I think this should be `"../../mix.lock.#{@target}"` as well, if we're going to continue to support ☔️  instead of just :poncho: projects. :trollface: